app/Config/Events.php

```
@@ -27,6 +28,11 @@
  */
 
 Events::on('pre_system', static function (): void {
+    $encryptionKey = config('Encryption')->key;
+    if (ENVIRONMENT !== 'testing' && (empty($encryptionKey) || strlen($encryptionKey) < 64)) {
```

PHPStan (2.2.7)

[error] Constant ENVIRONMENT not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)